src/core/domain/bring-instance-up-to-date-with-concept-snapshot-version-domain-service.ts

```diff
@@ -47,7 +47,10 @@ export class BringInstanceUpToDateWithConceptSnapshotVersionDomainService {
         const conceptSnapshotInInstanceLanguage = conceptSnapshot
             .transformLanguage(conceptSnapshotLanguage, instanceLanguage);
 
-        const instanceMergedWithConceptSnapshot = InstanceBuilder.from(instanceInStatusOntwerp)
+        const hasCompetentAuthorityLevelLokaal = instanceInStatusOntwerp.competentAuthorityLevels.includes(CompetentAuthorityLevelType.LOKAAL);
```

Comment:
**Question (non-blocking):** Is this check a deliberate addition to deal with incorrect concepts? More specifically, to deal with concepts that have a "local" authority level **and** have a competent authority filled in? A situation that should not occur if I understand the ticket's "more elaborate explanation" comment correctly.

Reply:
Yes it is. This test is not strictly necessary, given that the competent authorities should not be filled in for concepts about local authorities, but it is a fail-safe for when someone made an oopsie.
Maybe I could have left a little comment for that, but this code has very little comments over all and it is readable on its own.